Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -163,6 +163,7 @@ public struct KlaviyoSDK {
     ///   - deepLinkHandler: a completion handler that will be called when a notification contains a deep link.
     /// - Returns: true if the notificaiton originated from Klaviyo, false otherwise.
     public func handle(notificationResponse: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void, deepLinkHandler: ((URL) -> Void)? = nil) -> Bool {
+        KlaviyoBadgeCountUtil.syncBadgeCount()
```

Comment:
Does this need to happen here?

Reply:
It's an attempt to keep our count in sync with any other (non Klaviyo) notifications that go through this handler

Reply:
Yah I guess I'm asking if it could be done in the environment and as part of the action handling maybe for open event?